src/mol-canvas3d/canvas3d.ts

```diff
@@ -715,6 +746,10 @@ namespace Canvas3D {
 
         function draw(options?: { force?: boolean, xrFrame?: XRFrame }) {
             if (drawPaused || isContextLost) return;
+            if (!webgl.resources.finalizePrograms(requiredShaderVariants)) {
```

Comment:
If this ever returns false, it will break the animation (and possibly offscreen screenshot) rendering.

Can we pass `isSynchronous` from `tick()` and run this call with "force" parameter?